src/function/built_in_aggregate_functions.cpp

```diff
@@ -85,8 +85,11 @@ void BuiltInAggregateFunctions::registerCountStar() {
 void BuiltInAggregateFunctions::registerCount() {
     std::vector<std::unique_ptr<AggregateFunctionDefinition>> definitions;
     for (auto& typeID : DataType::getAllValidTypeIDs()) {
-        auto inputType =
-            typeID == LIST ? DataType(LIST, std::make_unique<DataType>(ANY)) : DataType(typeID);
+        auto inputType = typeID == common::TENSOR ?
```

Comment:
Do we need to support COUNT(TENSOR), I thought we will not support any operation on TENSOR yet but only read.

Reply:
Let's disable all functions on TENSOR (FIXED_LIST) for now.